src/css/skins/five.less

```diff
@@ -133,4 +131,24 @@
         border-radius: @five-border-radius;
     }
 
+    // override control bar element colors that don't look good in time slider above mode
+    &.jw-flag-time-slider-above,
+    &.jw-breakpoint-0,
+    &.jw-breakpoint-1 {
+      .jw-controlbar {
+        .jw-text {
```

Comment:
@robwalch nothing's wrong with it. I was asking if we could update the color var, which should have been a comment, not a review, but after talking with @madikarizma I understand why what I was thinking isn't possible. I'm ok with merging this.